head/src/main/kotlin/io/qalipsis/core/head/report/thymeleaf/ThymeleafReportServiceImpl.kt

```diff
@@ -167,20 +167,21 @@ internal class ThymeleafReportServiceImpl(
             require(html.isNotEmpty()) { "HTML File is empty" }
             val document = Jsoup.parse(html, CHARACTER_ENCODING)
             document.outputSettings().syntax(Document.OutputSettings.Syntax.xml)
+            val iTextRenderer = ITextRenderer()
             val outputStream = ByteArrayOutputStream()
             outputStream.use {
```

Comment:
Why outputStream.use {? you are not using it in the inside block.
My mistake for my earlier comment on the previous review, you should use
```
outputStream.use(iTextRenderer::createPDF)
```